### PR TITLE
Update the GameController message

### PR DESCRIPTION
**Pull request by ysims**
There were two new fields added, `numberOfWarnings` and `goalKeeper`. This adds those messages to the GameController message we are expecting.

### CODE REVIEW

**Review by tayloryoung6396 — COMMENTED** [review]
It looks good to me, will need some changes in GameController.cpp if we ever want to use it @Bidski

File: module/input/GameController/src/GameControllerData.hpp [hunk 1]
```diff
@@ -125,8 +127,10 @@ namespace module::input::gamecontroller {
     inline std::ostream& operator<<(std::ostream& os, const Robot& robot) {
         os << "\t\tPenalty state......: " << uint(robot.penaltyState) << std::endl
            << "\t\tPenalised time left: " << uint(robot.penalisedTimeLeft) << std::endl
+           << "\t\tNumber of warnings: " << uint(robot.numberOfWarnings) << std::endl
            << "\t\tYellow Card Count..: " << uint(robot.yellowCardCount) << std::endl
-           << "\t\tRed Card Count.....: " << uint(robot.redCardCount) << std::endl;
+           << "\t\tRed Card Count.....: " << uint(robot.redCardCount) << std::endl
+           << "\t\tGoalkeeper: " << uint(robot.goalKeeper) << std::endl;
```
Comment (tayloryoung6396) [inline module/input/GameController/src/GameControllerData.hpp:133]: this should probs be bool(robot.goalKeeper) or something

**Review by tayloryoung6396 — CHANGES_REQUESTED** [review]

File: module/input/GameController/src/GameControllerData.hpp (hunk 1 above)
Comment (ysims) [reply module/input/GameController/src/GameControllerData.hpp]: fixed the same way the other bool is done

**Review by tayloryoung6396 — APPROVED** [review]
Lgtm

**Pull request merged**